apps/src/gamelab/GameLab.js

```diff
@@ -75,6 +81,40 @@ var MEDIA_PROXY = '//' + location.host + '/media?u=';
 // starts with http or https
 var ABSOLUTE_REGEXP = new RegExp('^https?://', 'i');
 
+var ASSET_PATH_PREFIX = "/v3/assets/";
+
+/**
+ * If the filename is relative (contains no slashes), then prepend
+ * the path to the assets directory for this project to the filename.
+ *
+ * If the filename URL is absolute, route it through the MEDIA_PROXY.
+ * @param {string} filename
+ * @returns {string}
+ */
+GameLab.maybeAddAssetPathPrefix = function (filename) {
```

Comment:
As much as possible, can we extract this into common code?  It's not _very_ different from the same method in applab.js.